datanode/gateway/graphql/schema.graphql

```diff
@@ -856,19 +856,42 @@ type Epoch {
   ): DelegationsConnection
 }
 
+type NodeSet {
```

Comment:
Could you add in a line explaining what a node set is here? Also, I'm suggesting node set as 2 words below, but if you vehemently object, you can reject.

Reply:
I have no preference! Updated to "node set".